.github/workflows/check-pkg-availability.yml

```diff
@@ -0,0 +1,129 @@
+on:
+  workflow_call:
+    inputs:
+      module:
+        type: string
+        description: The module name
+        required: true
+      version:
+        type: string
+        description: The version
+        required: true
+    secrets:
+      registry_username:
+        required: true
+      registry_password:
+        required: true
+
+jobs:
+  check-stable-packages:
+    name: check-stable-packages ${{ matrix.distrib }}
+    runs-on: [ubuntu-22.04]
+    container:
+      image: ${{ vars.DOCKER_PROXY_REGISTRY_URL }}/${{ matrix.image }}
+      credentials:
+        username: ${{ secrets.registry_username }}
+        password: ${{ secrets.registry_password }}
+    strategy:
+      matrix:
+        include:
+        - image: 'almalinux:8'
+          distrib: el8
+        - image: 'almalinux:9'
+          distrib: el9
+        - image: 'debian:bullseye'
+          distrib: bullseye
+
+    steps:
+      - name: Checking stable package availability
+        run: |
+          set -eux
+
+          # Set version
+          MAJOR_VERSION=$(cut -d "." -f 1,2 <<< ${{ inputs.version }})
```

Comment:
why not use needs.get-version.outputs.major_version ?

Reply:
done